README.md

```diff
@@ -1,46 +1,111 @@
 # Verilator Verification Features Tests
 
+
 Copyright (c) 2022 [Antmicro](https://www.antmicro.com)
 
+
 This repository contains tests for a number of verification features in
 Verilator:
 
+
 * concurrent assertions,
 * functional coverage,
 * randomization constraints,
 * class parameters,
 * sequences,
 
+
 and more.
 
+
 ## Usage
 
-To run these tests, you need Robot Framework, PyYAML and Jinja2 installed on
-your system. `gen-tests` is a script that generates Robot test files for each
-test in the `tests` directory. To generate and run all tests:
+
+This repository contains several submodules that have their own submodules as
+well. In order to build properly, after cloning this repository, you shall run
+the following command within the repository directory (not necessary if you
+use the **scripts/build_and_run.sh** script for executing the flow):
+
+
+```
+git submodule update --init --recursive
+```
+
+
+### Requirements
+
+
+To run these tests, you need
+[Robot Framework](https://pypi.org/project/robotframework/),
+[PyYAML](https://pypi.org/project/PyYAML/), and
+[Jinja2](https://pypi.org/project/Jinja2/) installed on your system.
+
+
+### Automation
+
+
+To automate the process, you can use [act](https://github.com/nektos/act) as a
+workflow tool. Even though it is practical to execute, this flow would not
+generate the reports within the local repository directory. If you want to run
+the workflow manually and get the report automatically, you must first build the
+tools manually. In order to do so, execute the commands below when you are
+within the repository directory for running a basic script:
+
+
+```
+chmod +x scripts/build_and_run.sh
+```
+```
+./scripts/build_and_run.sh {OPTIONAL_BUILD_OPTS}
+```
+
+
+This script runs the git initialization, Verilator builds, test generation, and
+test run. If you would like to run specific commands within the flow, you may
+examine **.github/workflows/test.yml** or **scripts/build_and_run.sh**. In order
+to examine {OPTIONAL_BUILD_OPTS}, refer to the **Test Generation and Test Run**
+section below.
+
+
+### Test Generation and Test Run
+
+
+`gen-tests` is a script that generates Robot test files for each test in the
+`tests` directory. To generate and run all tests on an already-built environment,
+use:
+
 
 ```
 ./gen-tests
+```
+```
```

Comment:
So that they could be copied one by one through the button created on the right hand side.
![image](https://github.com/antmicro/verilator-verification-features-tests/assets/29541720/6b72ffaf-3d77-436f-ad90-d30c15304b52)

Reply:
Huh, never thought of that. Just FYI, you can paste multiple lines in a terminal and it'll execute them all. But if this approach is useful for you (maybe you want to do each step separately), then it's fine by me. :+1: